.gulp/gulpfile.iced

```diff
@@ -62,13 +62,13 @@ task 'test/nodejs-unit', 'run nodejs unit tests', [], (done) ->
   await execute "#{basefolder}/node_modules/.bin/mocha", defer _
   done();
 
-task 'testci/chrome-unit', '', [], (done) ->
+task 'testci/chrome-unit', '', ['testci/nodejs-unit'], (done) ->

```

Comment:
the CI jobs run the tests in parallel in different containers so I suggest not having a relationship between these tasks

Reply:
Removed in https://github.com/Azure/autorest.typescript/pull/211/commits/65d262eca39c20e4affd29ec9f696aa810c2cf41.